backend/app/services/adapters/bot_kinds.py

```
@@ -2001,20 +2019,21 @@ def find_available_bot_name(
-                    Kind.kind == "Bot",
-                    Kind.name == candidate,
-                    Kind.namespace == namespace,
-                    Kind.user_id == user_id,
-                    Kind.is_active == True,
-                )
-                .first()
+            query = db.query(Kind).filter(
+                Kind.kind == "Bot",
+                Kind.name == candidate,
+                Kind.namespace == namespace,
+                Kind.is_active == True,
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)